README.md

```diff
@@ -231,6 +231,42 @@ On all following examples we will assume a default user has been set in the conf
 We will also assume you are using `(:use starkbank.core)` on your code ns calls to
 expose the starkbank SDK namespaces.
 
+### 5. Resource listing and manual pagination
+
+Almost all SDK resources provide a `query` and a `page` function.
+
+- The `query` function provides a straight forward way to efficiently iterate through all results that match the filters you inform,
+seamlessly retrieving the next batch of elements from the API only when you reach the end of the current batch.
```

Comment:
não no clojure

Reply:
- The `query` function provides a straight forward way retrieve results that match the filters you inform,
seamlessly retrieving all the filtered elements from the API.